src/main/java/rx/Single.java

```diff
@@ -1569,14 +1569,12 @@ public final void onNext(T args) {
      * @param subscriber
      *            the Subscriber that will handle the emission or notification from the Single
      */
-    public final void unsafeSubscribe(Subscriber<? super T> subscriber) {
+    public final Subscription unsafeSubscribe(Subscriber<? super T> subscriber) {
```

Comment:
I strive to be as much as possible compatible with `Observable` type. Do you have some proposition how to handle it? If we remove `Subscription` as return type then we won't be able to decorate or replace the `Subscription` instance in hook.